### PR TITLE
create header

### LINKED ISSUE #2: Create header

### PR DESCRIPTION
**Pull request by JoannaTabbi**
Close #2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,21 @@
...
   <body>
-    <header></header>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="#">Hébergements</a></li>
+          <li><a href="#">Activités</a></li>
+        </ul>
+      </nav>
+      <div id="logo_titre">
```
Comment (Tutt-J) [inline index.html]: Inutil div

File: index.html [hunk 2]
```diff
@@ -11,7 +11,21 @@
...
+        <div id="logo">
```
Comment (Tutt-J) [inline index.html]: Inutil div

File: index.html [hunk 3]
```diff
@@ -11,7 +11,21 @@
...
+          <img src="assets/img/booki.png" alt="logo Booki" />
```
Comment (Tutt-J) [inline index.html]: Must be before nav

File: index.html [hunk 4]
```diff
@@ -11,7 +11,21 @@
...
+        </div>
+        <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
+        <p>En plein centre ville ou en pleine nature</p>
```
Comment (Tutt-J) [inline index.html]: Not in header, in search bar pr, rmeove it

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (JoannaTabbi) [reply index.html]: lines 25 and 26 removed

File: index.html (hunk 3 above)
Comment (JoannaTabbi) [reply index.html]: moved

File: index.html (hunk 2 above)
Comment (JoannaTabbi) [reply index.html]: div logo deletred

File: index.html (hunk 1 above)
Comment (JoannaTabbi) [reply index.html]: div logo_titre deleted

File: index.html [hunk 5]
```diff
@@ -11,7 +11,15 @@
     />
   </head>
   <body>
-    <header></header>
+    <header>
+      <img src="assets/img/booki.png" alt="logo Booki" />
```
Comment (Tutt-J) [inline index.html]: src="./assets/..." add ./

Comment (Tutt-J) [inline index.html]: Alt must be a phrase like "Logo de Booking"

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (JoannaTabbi) [reply index.html]: src corrected

Comment (JoannaTabbi) [reply index.html]: alt value modified

**Review by Tutt-J — APPROVED** [review]

**Pull request merged**